File: Algorithms/Implementation/Breaking The Records/breakingRecords.cs
//Problem URL: https://www.hackerrank.com/challenges/breaking-best-and-worst-records/problem

static int[] breakingRecords(int[] scores) {
    int min = scores[0], max = scores[0];
    int minBroken = 0, maxBroken = 0;

    for(int i = 0; i < scores.Length; i++){
        if(scores[i] > max){
            maxBroken += 1;
            max = scores[i];
        }
        else if(scores[i] < min){
            minBroken +=1;
            min = scores[i];
        }
    }

    return new int[]{maxBroken, minBroken};
}

File: Algorithms/Implementation/Breaking The Records/breakingRecords.py
#Problem URL: https://www.hackerrank.com/challenges/breaking-best-and-worst-records/problem

def breakingRecords(scores):
    min = scores[0]
    max = scores[0]
    minBroken = 0
    maxBroken = 0

    for i in range(0, len(scores)):
        if(scores[i] > max):
            maxBroken += 1
            max = scores[i]
        elif(scores[i] < min):
            minBroken += 1
            min = scores[i]
    
    return [maxBroken, minBroken]
